src/main/java/seedu/address/logic/commands/CreateGroupCommand.java

```diff
@@ -12,25 +15,32 @@
 import seedu.address.model.profbook.Group;
 
 /**
- * Represents a command for creating a new group within ProfBook.
- * This command is typically used to create a new group at a specified path.
+ * Represents a command for creating a new group in ProfBook.
  */
 public class CreateGroupCommand extends Command {
 
+    /** The command word for creating a group */
     public static final String COMMAND_WORD = "mkdir";
 
+    /** Error message indicating that a group with the provided ID already exists */
     public static final String MESSAGE_DUPLICATE_GROUP_ID =
             "GroupId %1$s has already been used by the group: %2$s";
 
+    /** Message indicating successful creating a new group */
     public static final String MESSAGE_SUCCESS = "New group added: %1$s";
 
+    /** A special instance of CreateGroupCommand used to display the command's usage message */
     public static final CreateGroupCommand HELP_MESSAGE = new CreateGroupCommand() {
```

Comment:
Do not repeat in comments information that is already obvious from the code. (refer to https://nus-cs2103-ay2324s1.github.io/website/se-book-adapted/chapters/codeQuality.html#do-not-repeat-the-obvious)